pkg/apis/core/validation/events.go

```diff
@@ -68,6 +67,9 @@ func ValidateEventCreate(event *core.Event, requestVersion schema.GroupVersion)
 	if event.Source.Component != "" || event.Source.Host != "" {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("source"), "", "needs to be unset"))
 	}
+	if event.Series != nil {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("series"), event.Series, "needs to be nil when an event is created"))
```

Comment:
Theoretically, we could allow the creation of Events with a Series but that wouldn't match the behavior defined in the KEP which doesn't sound correct. IMHO this validation should gate for correct utilization of the API by the clients.